tasks/main.yml

```diff
@@ -23,8 +23,8 @@
   pause:
     seconds: 10
 
-- name: "check clamdscan is working"
-  command: "clamdscan /bin/sh"
+#- name: "check clamdscan is working"
+#  command: "clamdscan /bin/sh"
 
```

Comment:
Why remove the "check clamdscan is working" task?

Reply:
That was an ad-hoc change at first, but it shouldn't have reached github.